<map id="pip._vendor.urllib3.exceptions.ConnectTimeoutError" name="pip._vendor.urllib3.exceptions.ConnectTimeoutError">
<area shape="rect" id="node5" href="$classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1NewConnectionError.html" title="pip._vendor.urllib3.exceptions.\lNewConnectionError" alt="" coords="5,348,204,389"/>
<area shape="rect" id="node2" href="$classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1TimeoutError.html" title="pip._vendor.urllib3.exceptions.\lTimeoutError" alt="" coords="5,169,204,211"/>
<area shape="rect" id="node3" href="$classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1HTTPError.html" title="pip._vendor.urllib3.exceptions.\lHTTPError" alt="" coords="5,80,204,121"/>
</map>
